aidants_connect_web/tests/test_models.py

```diff
@@ -109,3 +110,24 @@ def test_saving_and_retrieving_mandat(self):
             first_saved_item.perimeter, ["Carte grise", "Changement d'adresse"]
         )
         self.assertEqual(second_saved_item.usager.family_name, "Flanders")
+
+
+class UserModelTest(TestCase):
+    def test_what_happens_to_password_when_not_set(self):
```

Comment:
I'm not sure about these tests, it seems obvious that if you don't set attributes there won't be any, am I missing something?

Reply:
To me it is interesting to note in the tests that if an attribute is not set:
- it will not trigger an error
- it will translate to en empty string (and not `None` for example)

I find it is a way to document the code.

What do you think ?